static/js/message_edit.js

```diff
@@ -98,12 +98,9 @@ function is_widget_message(message) {
     return false;
 }
 
-export function get_editability(message, edit_limit_seconds_buffer = 0) {
+export function check_message_can_be_edited(message) {
```

Comment:
I think this is a good extraction but the name feels misleading; maybe `is_message_editable_ignoring_permissions` would be clear if a bit verbose?

I don't want random code paths to end up calling this function by accident.